packages/vue/src/components/reset-password.vue

```diff
@@ -8,14 +8,12 @@ import {
 import BaseFormFields from './primitives/base-form-fields.vue';
 
 import { useAuthenticator } from '../composables/useAuth';
-import { createSharedComposable } from '@vueuse/core';
 
 const attrs = useAttrs();
 const emit = defineEmits(['resetPasswordSubmit', 'backToSignInClicked']);
 
-const useAuthShared = createSharedComposable(useAuthenticator);
-const { send, submitForm } = useAuthShared();
-const { error, isPending } = toRefs(useAuthShared());
+const { send, submitForm } = useAuthenticator();
+const { error, isPending } = toRefs(useAuthenticator());
```

Comment:
Just curious, is the reason that we the separate calls to `useAuthenticator` here because `send` and `submitForm` maintain stable values whereas `error` and `isPending` are reactive?

Reply:
Yep, correct. We can reduce it down to one calls, but I'll put that in the subcomponents refactor!